src/Components/ArticlePage.jsx

```diff
@@ -3,45 +3,48 @@ import Button from 'react-bootstrap/Button'
 import {useParams, Link } from 'react-router'
 import {useEffect, useState}from 'react'
 import axios from 'axios'
-import CommentCard from './CommentCard'
+import CommentList from './CommentList';
 
 function ArticlePage (){
     const {article_id} = useParams();
     const [article, setArticle] = useState([])
-    const [comments, setComments] = useState([])
+    const [votes, setVotes] = useState(0)
     useEffect(() => {
         const getArticleById = async () => {
             const { data } = await axios.get(`https://nc-news-lo7q.onrender.com/api/articles/${article_id}`)
             setArticle(data.article)
+            setVotes(data.article.votes)
+            console.log(data.article)
+          
         }
         getArticleById()
     }, [article_id])
 
-    useEffect(() => {
-        const getCommentsByArticleId= async () => {
-            const { data } = await axios.get(`https://nc-news-lo7q.onrender.com/api/articles/${article_id}/comments`)
-            setComments(data.comments)
-        }
-        getCommentsByArticleId()
-    }, [article_id])
-
+    const patchVotes = async (increment) => {
+        await axios.patch(`https://nc-news-lo7q.onrender.com/api/articles/${article_id}`, {
+           inc_votes: increment,
+         })
+         console.log(article)
+         setVotes((votes) => votes + increment)
```

Comment:
It's good that you're setting votes optimistically here but a potential problem is that if the request to `patch` the votes fails then this state will need to be reverted back 💡 

This could be accompanied by some error messaging for a user to let them know there has been a problem

Reply:
& 🧹 ✨